odoo/addons/base/data/res_country_data.xml

```diff
@@ -1495,6 +1495,7 @@
             <field name="currency_id" ref="USD" />
             <field eval="1" name="phone_code" />
             <field name="vat_label"/> <!-- Empty to reset on existing DBs-->
+            <field name="clearing_number_label">ABA/Routing</field>
```

Comment:
Curious, are most countries have their own different `clearing_number_label`? If it is only for `ABA/Routing` used by US. Maybe leave the field in l10n_us, or hard code it in `_get_view`